src/palace.pyx

```diff
@@ -35,7 +35,8 @@ channel_configs : FrozenSet[str]
 __all__ = [
     'ALC_FALSE', 'ALC_TRUE', 'ALC_HRTF_SOFT', 'ALC_HRTF_ID_SOFT',
     'device_name_default', 'device_names', 'sample_types', 'channel_configs',
-    'sample_size', 'sample_length', 'query_extension', 'use_context',
+    'current_context', 'sample_size', 'sample_length',
+    'query_extension', 'use_context',
```

Comment:
Please sort these by functionality, i.e.
- sample_size, sample_length
- query_extension
- current_context, use_context

Line break where appropriate.

Reply:
Wait, should sample_type fall in the same line with sample_size and sample_length?

Reply:
`sample_types` and `channel_configs` are data, while the above are functions

Reply:
Hmm I think I've gotta sort the actual functions too.